app/assets/stylesheets/self-service-time-to-pay.css

```diff
@@ -7,6 +7,13 @@
     font-size: 48px;
 }
 
+.govuk-back-link, .print-summary-link {
+    display: none;
+}
+.js-enabled .govuk-back-link, .js-enabled .print-summary-link {
```

Comment:
since we're using the class `js-visible` would it be cleaner to implement it like we've done in the other services like this?
https://github.com/hmrc/self-assessment-refund-frontend/blob/main/app/assets/stylesheets/application.scss#L3

Reply:
Thanks for looking.

To implement your approach to the fix would mean a change to the back link component by adding a class to it (from a bit of digging/asking around, it appears to [sit here](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/backlink/BackLink.scala)). Do you think its worth making a change at a high level to that component, or sticking to my more localised fix?